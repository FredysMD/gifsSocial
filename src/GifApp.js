import React, { useState } from 'react';

import AddCategory from './components/AddCategory';
import GifGrid from './components/GifGrid';


const GifApp = () => {

    const [categories, setCategories] = useState(['']);

    return  (
        <div className = "container">
            <h2>GifSocial</h2>
            <AddCategory setCategories = { setCategories }/>
            <hr />
            <ol>
                {
                    categories.map( category => (
                        <GifGrid
                            key = { category }
                            category = { category } 
                        />
                    ))
                }
            </ol>
        </div>
    );
}

export default GifApp;