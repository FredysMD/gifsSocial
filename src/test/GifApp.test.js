import { shallow } from 'enzyme';
import React from 'react';

import  GifApp  from '../GifApp';


describe('Test to GifApp component', () => {
    
    test('should create snapShot', () => {
       
        const wrapper = shallow( <GifApp /> );

        expect( wrapper ).toMatchSnapshot();

    });
    

});

