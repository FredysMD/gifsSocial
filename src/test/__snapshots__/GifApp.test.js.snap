// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test to GifApp component should create snapShot 1`] = `
<div
  className="container"
>
  <h2>
    GifSocial
  </h2>
  <AddCategory
    setCategories={[Function]}
  />
  <hr />
  <ol>
    <GifGrid
      category=""
      key=""
    />
  </ol>
</div>
`;
